src/components/shared/__tests__/Property.spec.tsx

```diff
@@ -64,7 +64,7 @@ describe('Property component', () => {
 
     const wrapper = render(schema);
     expect(wrapper.find(Types)).toHaveHTML(
-      '<span class="text-green-6 dark:text-green-4 truncate">array[string]</span>',
+      '<span class="sl-truncate sl-text-muted">array[string]</span>',
```

Comment:
Have I mentioned that I hate snapshot tests yet? :trollface:

Reply:
Constantly. And I agree :)